core/arch/arm/kernel/spmc_sp_handler.c

```diff
@@ -149,7 +149,71 @@ static int spmc_sp_add_sp_region(struct sp_mem *smem,
 				 struct sp_session *owner_sp,
 				 uint8_t highest_permission)
 {
+	struct sp_ctx *sp_ctx = NULL;
+	uint64_t va = READ_ONCE(mem_reg->address);
+	int res = FFA_OK;
+	uint64_t region_len = READ_ONCE(mem_reg->page_count) * SMALL_PAGE_SIZE;
+	struct mobj *mobj = NULL;
+
+	sp_ctx = to_sp_ctx(owner_sp->ts_sess.ctx);
+
+	/*
+	 * The memory region we try to share might not be linked to just 1
+	 * mobj. Create a new region for each mobj.
+	 */
+	while (region_len) {
+		uint64_t len = region_len;
+		struct sp_mem_map_region *region = NULL;
+		uint16_t prot = 0;
+		uint64_t offs = 0;
+
+		/*
+		 * There is already a mobj for each address that is in the SPs
+		 * address range.
+		 */
+		mobj = vm_get_mobj(&sp_ctx->uctx, va, &len,
```

Comment:
`vm_get_mobj()` expects `&len` and `&offs` to be pointers to `size_t` instead of `uint64_t`.

Reply:
Done